Cargo.toml

```diff
@@ -73,10 +73,11 @@ edition = "2021"
 license = "Apache-2.0"
 
 [workspace.lints]
-clippy.print_stdout = "warn"
-clippy.print_stderr = "warn"
 clippy.dbg_macro = "warn"
 clippy.implicit_clone = "warn"
+clippy.result_large_err = "allow"
+clippy.large_enum_variant = "allow"
```

Comment:
these two are generally considered bad for performance, why allow them?

Reply:
These are reported right after changing the toolchain version. 
After a brief look, I found that there are too many places to change to fix these two. We can fix them in a separate PR later.

Reply:
That's fine. FYI if the large err is caused by snafu, you can simply do like [this](https://github.com/GreptimeTeam/greptimedb-edge-ingester-java/blob/47f823d29567013e42d3c94515757e2b9d575fa4/core/src/main/rust/ingester/src/error.rs#L26)